### PR TITLE
Prevent double plugin_load

### PR DESCRIPTION
**Pull request by lipsanen**

### CODE REVIEW

File: spt/spt-serverplugin.cpp [hunk 1]
```diff
@@ -262,6 +271,13 @@ bool CSourcePauseTool::Load(CreateInterfaceFn interfaceFactory, CreateInterfaceF
...
 void CSourcePauseTool::Unload(void)
 {
+	if (skipUnload)
+	{
+		// Preventing double load of plugin, do nothing on unload
+		skipUnload = false; // Enable unloading again
+		return;
+	}
+
 #if !defined(OE)
```
Comment (YaLTeR) [inline spt/spt-serverplugin.cpp:281]: Should this also set pluginLoaded = false?

Comment (lipsanen) [reply spt/spt-serverplugin.cpp:281]: No, when this occurs the plugin is still loaded and is running on the same instance of that DLL if that makes sense.

Comment (YaLTeR) [reply spt/spt-serverplugin.cpp:281]: I mean when you unload the last copy.

Comment (lipsanen) [reply spt/spt-serverplugin.cpp:281]: It still unloads and loads fine, in a real unload the library gets freed completely it seems.
image

Comment (YaLTeR) [reply spt/spt-serverplugin.cpp:281]: ah, cool

**Pull request merged**